### PR TITLE
Fix some CHANGELOG entries

### PR DESCRIPTION
**Pull request by tiborvass**
Signed-off-by: Tibor Vass tibor@docker.com

### CODE REVIEW

**Comment by aaronlehmann** [comment]
LGTM

**Comment by icecrime** [comment]
LGTM

File: CHANGELOG.md [hunk 1]
```diff
@@ -143,12 +142,16 @@ be found.
...
+- Fix a panic during cleanup if a container was started with invalid options ([#21802](https://github.com/docker/docker/pull/21802))
+- Fix a situation where a container cannot be stopped if the terminal is closed ([#21840](https://github.com/docker/docker/pull/21840))
 
 ### Security
 
 * Object with the `pcp_pmcd_t` selinux type were given management access to `/var/lib/docker(/.*)?` ([#21370](https://github.com/docker/docker/pull/21370))
 * `restart_syscall`, `copy_file_range`, `mlock2` joined the list of allowed calls in the default seccomp profile ([#21117](https://github.com/docker/docker/pull/21117), [#21262](https://github.com/docker/docker/pull/21262))
 * `send`, `recv` and `x32` were added to the list of allowed syscalls and arch in the default seccomp profile ([#19432](https://github.com/docker/docker/pull/19432))
+* Docker Content Trust now requests the server to perform snapshot signing ([#21046](https://github.com/docker/docker/pull/21046))
+* Support for using Yubikeys for Content Trust signing has been moved out of experimental ([#21591](https://github.com/docker/docker/pull/21591))
```
Comment (thaJeztah) [inline CHANGELOG.md]: nit s/Yubikeys/YubiKeys/

(But "meh")

**Comment by thaJeztah** [comment]
LGTM

**Comment by tiborvass** [comment]
@thaJeztah fixed

**Comment by NathanMcCauley** [comment]
LGTM

**Pull request merged**